locals.tf

```diff
@@ -35,12 +35,32 @@ locals {
 
   cool_shared_services_cidr_block = data.terraform_remote_state.sharedservices_networking.outputs.vpc.cidr_block
 
-  guacamole_fqdn = format("guac.%s.%s", var.assessment_account_name, var.cool_domain)
+  guacamole_fqdn = format("guac.%s.%s", local.assessment_account_name_base, var.cool_domain)
 
-  # Find the Images account by name.
+  # Look up assessment account name from AWS organizations provider
+  assessment_account_name = [
+    for account in data.aws_organizations_organization.cool.accounts :
+    account.name
+    if account.id == local.assessment_account_id
+  ][0]
+
+  # Determine assessment account type based on account name.
+  #
+  # The account name format is "ACCOUNT_NAME (ACCOUNT_TYPE)" - for
+  # example, "Shared Services (Production)".
+  assessment_account_type = length(regexall("\\(([^()]*)\\)", local.assessment_account_name)) == 1 ? regex("\\(([^()]*)\\)", local.assessment_account_name)[0] : "Unknown"
+  workspace_type          = lower(local.assessment_account_type)
+
+  # The Terraform workspace name for this assessment
+  assessment_workspace_name = replace(replace(lower(var.assessment_account_name), "/[()]/", ""), " ", "-")
+
+  assessment_account_name_base = split(" ", var.assessment_account_name)[0]
```

Comment:
We should probably add a comment here that mentions the assumption that our assessment account names are a single word followed by the account type, e.g. "env0 (Production)".

Reply:
See commit b1dc956.

Reply:
See also commit 10cb0a2, which makes the assumption only that there not be a second "(" in the name.